MonoGameGum/KniGum.csproj

```diff
@@ -0,0 +1,100 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <PackageId>Gum.KNI</PackageId>
+    <Version>2024.6.20.2</Version>
+
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+
+
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>$(DefineConstants);MONOGAME;KNI;USE_GUMCOMMON</DefineConstants>
```

Comment:
Here I define both MONOGAME and KNI. Perhaps it's best to define only KNI and expand all 
`#if MONOGAME` 
to
`#if MONOGAME || KNI`

Reply:
Ya I think that's probably best, even though it's more work.

Reply:
Done.